.github/workflows/ci-unix-shared-installed.yml

```diff
@@ -0,0 +1,68 @@
+# This is a copy of ci-unix-shared-local.yml for building shared libraries, but
+# using installed dependencies instead of locally built ones.
+
+name: CI
+on: [push, pull_request]
+jobs:
+  build-shared-installed:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set GCC & G++ 10 compiler (on Linux)
+      if: runner.os == 'Linux'
+      run: echo "CC=gcc-10" >> $GITHUB_ENV && echo "CXX=g++-10" >> $GITHUB_ENV
+
+    - name: Cache external dependencies
+      id: cache-ext
+      uses: actions/cache@v3
+      with:
+        path: ext
+        key: ${{ runner.os }}-shared-installed-${{ hashFiles('ext/*.cmd') }}
+    - name: Setup cmake
+      uses: jwlawson/actions-setup-cmake@v1.12
+      with:
+        cmake-version: '3.13.x'
+    - name: Print cmake version
+      run: cmake --version
+    - uses: seanmiddleditch/gha-setup-ninja@v3
+    - name: Set shared libs
+      if: steps.cache-ext.outputs.cache-hit != 'true'
+      working-directory: ./ext
+      run: sed -i'' -e 's/-DBUILD_SHARED_LIBS=OFF/-DBUILD_SHARED_LIBS=ON/' *.cmd
+    - name: Add recent repository for libyuv-dev
+      if: steps.cache-ext.outputs.cache-hit != 'true'
```

Comment:
Remove this `if` statement, because the side effects (output files) of the run statement (line 39) are not in the cache files. Note that line 24 says the cache files only include the `ext` directory.

For the same reason, remove the `if` statements at line 41 and line 44.

Do keep the `if` statement at line 50 because GoogleTest build output files are in the `ext` directory.